app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -108,20 +108,11 @@ class Contact extends React.Component {
 
     let current = this.state.current;
     let modified = this.state.modified;
-    const updates = { [event.target.id]: value };
```

Comment:
more for the sake of conflict for the alternate contact stuff, can you leave this updates variable as is and just remove the code below?  I still need to use updates as a part of that PR

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1106/commits/ce6a5a2e27cd5120050ddef6601ebcc7abbcaab5